include/enums.h

```diff
@@ -237,7 +237,7 @@ typedef UNK_TYPE Easing;
 #define Easing_SIN_OUT            0x0000000B
 #define Easing_COS_IN             0x0000000C
 
-typedef UNK_TYPE SoundId;
+typedef UNK_TYPE SoundID;
```

Comment:
Good, we should rename the below defines too that are in the `SoundId` namespace